electrode-ota-server-model-acquisition/test/acquisition-test.js

```diff
@@ -410,15 +412,69 @@ describe('model/acquisition', function () {
         });
     });
 
+    describe("updateCheck with --targetBinaryVersion in semver", () => {
+        const email = 'test@unit-test.com';
+        const name = 'TestSemverApp';
+        let stagingKey = '';
+        let productionKey = '';
+        let clientUniqueId = '190jf09j2f01j10901';
+
+        before(() => {
+            return appBL.createApp({ email, name }).then(a => {
+                return dao.deploymentByApp(a.id, 'Staging').then(deployment => {
+                    stagingKey = deployment.key;
+                });
+            });
+        });
+
+        it("should target any minor version", () => {
+            return appBL
+              .upload({
```

Comment:
can u check if the upload API endpoint works with semver?

Reply:
the `appBL` here is the `electrode-ota-server-model-app` package that the upload API endpoint uses. If needed we can add more tests for the same package.